src/System.Numerics.Vectors/ref/Configurations.props

```diff
@@ -1,16 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard1.0;
-      net45;
-      netstandard;
-      net46;
-    </PackageConfigurations>
     <BuildConfigurations>
       netcoreapp;
+      netstandard;
```

Comment:
netstandard should not be needed here.

Reply:
I thought it was needed since it is "in box" for netstandard2.1?

Reply:
The moment you stop building the package, you just need to care about current runtime version only. No need to worry about netstandard, etc.

Reply:
Correct, netstandard2.1 is going to be handled with https://github.com/dotnet/corefx/issues/36719

Reply:
You *may* need the `netstandard` configuration if we have some other library with a `netstandard` configuration that needs this DLL.  We don't want to restore it because that will break source build.  IOW, remove `netstandard` configuration and if it the build fails, add it back with a comment explaining why its needed.